Training Modules/Introduction to Numpy/(jupyter notebook)Introduction to Numpy.ipynb

```diff
@@ -0,0 +1,1034 @@
+{
```

Comment:
<p>Please remove the unnecessary bold on all headings.</p><p>Please follow the Notebook guidelines mentioned in #12 .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>